### PR TITLE
chore: merge up to 0.83-stable branch

### PR DESCRIPTION
**Pull request by gabrieldonadel**
## Summary:

Merge up to the first commit on `0.83-stable` branch from upstream (a034841fd68a93778d4999f8bc540cf6a5fef8d4).

<details>
<summary>Full list files where conflicts had to be manually resolved</summary>

- README.md
- package.json
- packages/assets/package.json
- packages/babel-plugin-codegen/package.json
- packages/community-cli-plugin/package.json
- packages/core-cli-utils/package.json
- packages/debugger-frontend/package.json
- packages/debugger-shell/package.json
- packages/dev-middleware/package.json
- packages/eslint-config-react-native/package.json
- packages/eslint-plugin-react-native/package.json
- packages/eslint-plugin-specs/package.json
- packages/gradle-plugin/package.json
- packages/metro-config/package.json
- packages/new-app-screen/package.json
- packages/normalize-color/package.json
- packages/polyfills/package.json
- packages/react-native-babel-preset/package.json
- packages/react-native-babel-transformer/package.json
- packages/react-native-codegen/package.json
- packages/react-native-compatibility-check/package.json
- packages/react-native-popup-menu-android/package.json
- packages/react-native-test-library/ios/RCTSampleNativeComponentComponentView.mm
- packages/react-native-test-library/ios/RCTSampleNativeComponentViewManager.mm
- packages/react-native-test-library/package.json
- packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h
- packages/react-native/Libraries/AppDelegate/RCTAppSetupUtils.h
- packages/react-native/Libraries/AppDelegate/RCTAppSetupUtils.mm
- packages/react-native/Libraries/AppDelegate/RCTDefaultReactNativeFactoryDelegate.mm
- packages/react-native/Libraries/AppDelegate/RCTReactNativeFactory.h
- packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.h
- packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.mm
- packages/react-native/Libraries/Components/Pressable/Pressable.js
- packages/react-native/Libraries/Components/TextInput/TextInput.flow.js
- packages/react-native/Libraries/Components/TextInput/TextInputState.js
- packages/react-native/Libraries/Components/View/View.js
- packages/react-native/Libraries/Components/View/ViewNativeComponent.js
- packages/react-native/Libraries/Image/Image.ios.js
- packages/react-native/Libraries/Image/RCTBundleAssetImageLoader.mm
- packages/react-native/Libraries/Image/RCTImageBlurUtils.mm
- packages/react-native/Libraries/Image/RCTImageLoader.mm
- packages/react-native/Libraries/Image/RCTImageViewManager.mm
- packages/react-native/Libraries/Image/RCTLocalAssetImageLoader.mm
- packages/react-native/Libraries/Image/RCTUIImageViewAnimated.mm
- packages/react-native/Libraries/LinkingIOS/RCTLinkingManager.mm
- packages/react-native/Libraries/NativeAnimation/Nodes/RCTInterpolationAnimatedNode.mm
- packages/react-native/Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.mm
- packages/react-native/Libraries/Text/Text.js
- packages/react-native/Libraries/Text/Text/RCTTextView.h
- packages/react-native/Libraries/Text/Text/RCTTextView.mm
- packages/react-native/Libraries/Text/TextInput/RCTBaseTextInputView.h
- packages/react-native/Libraries/Text/TextInput/RCTInputAccessoryView.h
- packages/react-native/Libraries/Text/TextInput/RCTInputAccessoryViewContent.h
- packages/react-native/Libraries/Text/VirtualText/RCTVirtualTextView.h
- packages/react-native/Libraries/Utilities/__tests__/Platform-test.js
- packages/react-native/Libraries/Wrapper/Example/RCTWrapperReactRootViewController.h
- packages/react-native/Libraries/Wrapper/RCTWrapper.h
- packages/react-native/Libraries/Wrapper/RCTWrapperView.h
- packages/react-native/Libraries/Wrapper/RCTWrapperViewControllerHostingView.h
- packages/react-native/React/Base/RCTBridgeProxy.mm
- packages/react-native/React/Base/RCTRootView.h
- packages/react-native/React/CoreModules/RCTActionSheetManager.mm
- packages/react-native/React/CoreModules/RCTDevLoadingView.mm
- packages/react-native/React/CoreModules/RCTDevMenu.h
- packages/react-native/React/CoreModules/RCTDevMenu.mm
- packages/react-native/React/CoreModules/RCTDeviceInfo.mm
- packages/react-native/React/CoreModules/RCTFPSGraph.mm
- packages/react-native/React/CoreModules/RCTLogBoxView.h
- packages/react-native/React/CoreModules/RCTLogBoxView.mm
- packages/react-native/React/CoreModules/RCTPerfMonitor.mm
- packages/react-native/React/DevSupport/RCTInspectorDevServerHelper.mm
- packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTPullToRefreshViewComponentView.mm
- packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm
- packages/react-native/React/Fabric/Mounting/ComponentViews/Text/RCTParagraphComponentView.mm
- packages/react-native/React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm
- packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewAccessibilityElement.h
- packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewAccessibilityElement.mm
- packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm
- packages/react-native/React/Fabric/Mounting/RCTComponentViewDescriptor.h
- packages/react-native/React/Fabric/RCTSurfacePointerHandler.mm
- packages/react-native/React/Fabric/RCTSurfaceTouchHandler.mm
- packages/react-native/React/Fabric/Utils/RCTBoxShadow.mm
- packages/react-native/React/Modules/RCTUIManager.h
- packages/react-native/React/Modules/RCTUIManager.mm
- packages/react-native/React/Views/RCTActivityIndicatorView.m
- packages/react-native/React/Views/RCTModalHostView.h
- packages/react-native/React/Views/RCTModalHostView.m
- packages/react-native/React/Views/RCTModalHostViewController.h
- packages/react-native/React/Views/RCTModalHostViewController.m
- packages/react-native/React/Views/RCTModalHostViewManager.h
- packages/react-native/React/Views/RCTModalHostViewManager.m
- packages/react-native/React/Views/RCTSwitch.h
- packages/react-native/React/Views/RefreshControl/RCTRefreshControl.h
- packages/react-native/React/Views/RefreshControl/RCTRefreshControl.m
- packages/react-native/React/Views/RefreshControl/RCTRefreshControlManager.m
- packages/react-native/React/Views/SafeAreaView/RCTSafeAreaView.m
- packages/react-native/React/Views/SafeAreaView/RCTSafeAreaViewManager.m
- packages/react-native/React/Views/ScrollView/RCTScrollView.h
- packages/react-native/React/Views/ScrollView/RCTScrollView.m
- packages/react-native/React/Views/ScrollView/RCTScrollViewManager.h
- packages/react-native/ReactCommon/react/renderer/components/textinput/TextInputEventEmitter.h
- packages/react-native/ReactCommon/react/renderer/graphics/platform/ios/react/renderer/graphics/HostPlatformColor.mm
- packages/react-native/ReactCommon/react/renderer/graphics/platform/ios/react/renderer/graphics/RCTPlatformColorUtils.h
- packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/ios/react/renderer/textlayoutmanager/RCTFontUtils.h
- packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/ios/react/renderer/textlayoutmanager/RCTFontUtils.mm
- packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/ios/react/renderer/textlayoutmanager/RCTTextLayoutManager.mm
- packages/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTHermesInstance.h
- packages/react-native/package.json
- packages/react-native/sdks/hermes-engine/hermes-engine.podspec
- packages/react-native/sdks/hermes-engine/hermes-utils.rb
- packages/rn-tester/NativeComponentExample/ios/RNTMyLegacyNativeViewManager.mm
- packages/rn-tester/NativeComponentExample/ios/RNTMyNativeViewManager.mm
- packages/rn-tester/Podfile.lock
- packages/rn-tester/RNTester/AppDelegate.mm
- packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj
- packages/rn-tester/js/examples/Cursor/CursorExample.js
- packages/rn-tester/js/examples/PlatformColor/PlatformColorExample.js
- packages/rn-tester/package.json
- packages/typescript-config/package.json
- packages/virtualized-lists/package.json
- private/helloworld/Gemfile
- private/helloworld/Gemfile.lock
- private/helloworld/package.json
- private/react-native-codegen-typescript-test/package.json
- yarn.lock

</details>

## Test Plan:

Run RNTester locally

image

### CODE REVIEW

File: packages/dev-middleware/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "node": ">= 20.19.4"
   },
   "devDependencies": {
-    "@react-native/debugger-shell": "0.83.0-main",
+    "@react-native/debugger-shell": "workspace:*",
```
Comment (Saadnajmi) [inline packages/dev-middleware/package.json:42]: 🚀

File: packages/react-native/Libraries/Components/View/View.js [hunk 2]
```diff
@@ -157,6 +157,14 @@ component View(
...
     };
   }
+  // [macOS
+  if (otherProps.onKeyDown) {
+    processedProps.onKeyDown = _onKeyDown;
+  }
+  if (otherProps.onKeyUp) {
+    processedProps.onKeyUp = _onKeyUp;
+  }
+  // macOS]
```
Comment (Saadnajmi) [inline packages/react-native/Libraries/Components/View/View.js:167]: Why was this added? Was it not there before o_0

File: packages/react-native/React/Base/RCTUtils.mm [hunk 3]
```diff
@@ -451,7 +451,11 @@ CGSize RCTSwitchSize(void)
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
     RCTUnsafeExecuteOnMainQueueSync(^{
+#if !TARGET_OS_OSX // [macOS]
       rctSwitchSize = [UISwitch new].intrinsicContentSize;
+#else // [macOS
+      rctSwitchSize = [NSSwitch new].intrinsicContentSize;
+#endif // macOS]
```
Comment (Saadnajmi) [inline packages/react-native/React/Base/RCTUtils.mm]: nit: might as well just add a `RCTPlatformSwitch` typedef?

File: packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm [hunk 4]
```diff
@@ -696,8 +698,12 @@ - (void)_updateStateWithContentOffset
         }
         auto newData = oldData;
         newData.contentOffset = contentOffset;
+#if !TARGET_OS_OSX // [macOS]
         newData.disableViewCulling =
             UIAccessibilityIsVoiceOverRunning() || UIAccessibilityIsSwitchControlRunning() || isAccessibilityAPIUsed;
+#else // [macOS
+        newData.disableViewCulling = isAccessibilityAPIUsed;
+#endif // macOS]
```
Comment (Saadnajmi) [inline packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm:772]: Do we need to implement this? It looks like we won't have view culling without it, and there are some macOS equivalents (Claude tells me NSWorkspace.shared.isVoiceOverEnabled (or NSAccessibility.isVoiceOverRunning on older SDKs), NSWorkspace.accessibilityDisplayOptionsDidChangeNotification for change notifications).

for `accessibilityElementCount` and `focusItemsInRect`, I don't think there are easy equivalents, but I haven't looked how they are used elsewhere.

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm [hunk 5]
```diff
@@ -1871,19 +1858,6 @@ - (void)drawFocusRingMask
...
-- (void)focus
-{
-  [[self window] makeFirstResponder:self];
-}
-
-- (void)blur
-{
-  [[self window] resignFirstResponder];
-}
-
```
Comment (Saadnajmi) [inline packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm:1767]: Did you notice whether using `ref.current.focus()`  got affected at all after this?

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm [hunk 6]
```diff
@@ -1895,32 +1869,6 @@ - (BOOL)acceptsFirstResponder
...
-    return NO;
-  }
-
-  if (_eventEmitter) {
-    _eventEmitter->onBlur();
-  }
-
-  return YES;
-}
-
```
Comment (Saadnajmi) [inline packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm:1804]: These were for iOS compatibility, why where they removed?

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm [hunk 7]
```diff
@@ -2369,7 +2317,9 @@ - (void)transferVisualPropertiesFromView:(UIView *)sourceView toView:(UIView *)d
   // corner
   destinationView.layer.cornerRadius = sourceView.layer.cornerRadius;
   sourceView.layer.cornerRadius = 0;
+#if !TARGET_OS_OSX // [macOS]
   destinationView.layer.cornerCurve = sourceView.layer.cornerCurve;
+#endif // [macOS]
```
Comment (Saadnajmi) [inline packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm]: This is on Appkit too? https://developer.apple.com/documentation/quartzcore/calayer/cornercurve

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm [hunk 8]
```diff
@@ -2409,11 +2359,17 @@ - (void)handleCommand:(const NSString *)commandName args:(const NSArray *)args
...
   [self becomeFirstResponder];
 }
 
 - (void)blur
 {
+#if TARGET_OS_OSX // [macOS
+  [[self window] resignFirstResponder];
+#endif // macOS]
   [self resignFirstResponder];
 }
```
Comment (Saadnajmi) [inline packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm:2486]: Why did these move? I guess because we removed the iOS check? But the implementation is different (We aren't checkig the return of super)

File: packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift [hunk 9]
```diff
@@ -6,30 +6,49 @@
...
+
+#if os(macOS) // [macOS
+public typealias RCTPlatformView = NSView
+public typealias RCTUIColor = NSColor
+public typealias RCTPlatformHostingController = NSHostingController
+#else
+public typealias RCTPlatformView = UIView
+public typealias RCTUIColor = UIColor
+public typealias RCTPlatformHostingController = UIHostingController
+#endif // macOS]
```
Comment (Saadnajmi) [inline packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift:23]: Ooooh do we need RCTUIKit.swift now :D

Comment (Saadnajmi) [reply packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift:23]: (This is fine to keep here for now)

File: packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift [hunk 10]
```diff
@@ -6,30 +6,49 @@
  */
 
 import SwiftUI
+#if os(macOS) // [macOS
+import AppKit
+#else
 import UIKit
+#endif // macOS]
```
Comment (Saadnajmi) [inline packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift:13]: Is RCTUIKit not importable? Maybe a followup for me, but is it worth breaking that out into it's own thing finally?

File: packages/react-native/scripts/codegen/generate-artifacts-executor/utils.js [hunk 11]
```diff
@@ -463,7 +463,7 @@ function findDisabledLibrariesByPlatform(
 
 function findReactNativeRootPath(projectRoot /* : string */) /* : string */ {
   const reactNativePackageJsonPath = require.resolve(
-    path.join('react-native', 'package.json'),
+    path.join('react-native-macos', 'package.json'), // [macOS]
```
Comment (Saadnajmi) [inline packages/react-native/scripts/codegen/generate-artifacts-executor/utils.js:466]: should this use a CONST for package name? I've done that before (and pushed upstream)

File: packages/react-native/scripts/featureflags/ReactNativeFeatureFlags.config.js [hunk 12]
```diff
@@ -347,7 +347,7 @@ const definitions: FeatureFlagDefinitions = {
       ossReleaseStage: 'none',
     },
     enableImperativeFocus: {
-      defaultValue: false,
+      defaultValue: true, // [macOS]
```
Comment (Saadnajmi) [inline packages/react-native/scripts/featureflags/ReactNativeFeatureFlags.config.js:350]: 🎉

File: packages/react-native/sdks/hermes-engine/hermes-engine.podspec [hunk 13]
```diff
@@ -26,8 +26,7 @@ versionProperties = Hash[*File.read("version.properties").split(/[=\n]+/)]
 if ENV['RCT_HERMES_V1_ENABLED'] == "1"
   version = versionProperties['HERMES_V1_VERSION_NAME']
 else
-  version = findMatchingHermesVersion(package) # [macOS] Use special logic instead of just package['version']
-  # version = versionProperties['HERMES_VERSION_NAME']
+  version = versionProperties['HERMES_VERSION_NAME']
```
Comment (Saadnajmi) [inline packages/react-native/sdks/hermes-engine/hermes-engine.podspec:29]: Why this change? Does our helper `findMatchingHermesVersion` fail?

File: packages/react-native/sdks/hermes-engine/hermes-utils.rb [hunk 14]
```diff
@@ -336,23 +336,6 @@ def resolve_url_redirects(url)
...
-        return nil
-    end
-
-    if package['peerDependencies']
-        return package['peerDependencies']['react-native']
-    end
-
-    hermes_log("No matching Hermes version found. Defaulting to main branch, which may be unreliable.")
-end
-# macOS]
```
Comment (Saadnajmi) [inline packages/react-native/sdks/hermes-engine/hermes-utils.rb:332]: Same comment as above

File: packages/react-native/sdks/hermes-engine/version.properties [hunk 15]
```diff
@@ -1,2 +1,2 @@
-HERMES_VERSION_NAME=1000.0.0
-HERMES_V1_VERSION_NAME=250829098.0.1
+HERMES_VERSION_NAME=0.14.0
```
Comment (Saadnajmi) [inline packages/react-native/sdks/hermes-engine/version.properties:1]: We will probably want to undo this, and the `.hermesversion` commits unless the lookup fails for some reason.

File: packages/react-native/package.json [hunk 16]
```diff
@@ -178,7 +178,7 @@
     "commander": "^12.0.0",
     "flow-enums-runtime": "^0.0.6",
     "glob": "^7.1.1",
-    "hermes-compiler": "0.0.0",
+    "hermes-compiler": "0.14.0",
```
Comment (Saadnajmi) [inline packages/react-native/package.json:202]: Same comment as above

File: packages/rn-tester/js/examples/Cursor/CursorExample.js [hunk 17]
```diff
@@ -8,8 +8,8 @@
  * @format
  */
 
-import type {CursorValue} from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
 import type {RNTesterModuleExample} from '../../types/RNTesterTypes';
+import type {CursorValue} from 'react-native/Libraries/StyleSheet/StyleSheetTypes';
```
Comment (Saadnajmi) [inline packages/rn-tester/js/examples/Cursor/CursorExample.js:12]: Undo?

File: packages/rn-tester/js/examples/Snapshot/SnapshotExample.js [hunk 18]
```diff
@@ -44,7 +44,7 @@ class ScreenshotExample extends React.Component<{...}, $FlowFixMe> {
 
   // [macOS] alert needs two string arguments, passing an error results in crashing
   takeScreenshot = () => {
-    if (ScreenshotManager !== undefined && ScreenshotManager !== null) {
+    if (ScreenshotManager !== undefined) {
```
Comment (Saadnajmi) [inline packages/rn-tester/js/examples/Snapshot/SnapshotExample.js:47]: I guess it's always nonnull?

**Review by Saadnajmi — CHANGES_REQUESTED** [review]

File: packages/react-native/Libraries/Components/View/View.js (hunk 2 above)
Comment (gabrieldonadel) [reply packages/react-native/Libraries/Components/View/View.js:167]: This is just using the same `processedProps` pattern as all props in View.js. This PR adds some more context https://github.com/facebook/react-native/pull/51225

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm (hunk 5 above)
Comment (gabrieldonadel) [reply packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm:1767]: This is still implemented, it just moved around because it was added on core. https://github.com/facebook/react-native/issues/53825

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm (hunk 6 above)
Comment (gabrieldonadel) [reply packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm:1804]: They were added to core, this just removes the code duplication 

https://github.com/facebook/react-native/issues/53827

File: packages/rn-tester/js/examples/Snapshot/SnapshotExample.js (hunk 18 above)
Comment (gabrieldonadel) [reply packages/rn-tester/js/examples/Snapshot/SnapshotExample.js:47]: Yes, flow was complaning about it

File: packages/rn-tester/js/examples/Cursor/CursorExample.js (hunk 17 above)
Comment (gabrieldonadel) [reply packages/rn-tester/js/examples/Cursor/CursorExample.js:12]: We need to add RNTesterModuleExample to use `] as Array<RNTesterModuleExample>;` otherwise flow will error out

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm (hunk 8 above)
Comment (gabrieldonadel) [reply packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm:2486]: This changed slightly because it was implemented on core, it tried to keep the same functionality and the code close to what we had before

File: packages/react-native/scripts/codegen/generate-artifacts-executor/utils.js (hunk 11 above)
Comment (gabrieldonadel) [reply packages/react-native/scripts/codegen/generate-artifacts-executor/utils.js:466]: Something like this? I can follow up on core if that's the case  

https://github.com/microsoft/react-native-macos/blob/0bb9e8952ada058a821c972753008aa74752389d/packages/react-native/react-native.config.js#L186

File: packages/react-native/sdks/hermes-engine/hermes-engine.podspec (hunk 13 above)
Comment (gabrieldonadel) [reply packages/react-native/sdks/hermes-engine/hermes-engine.podspec:29]: This was just realigned with core now that we no longer build from source. Also `findMatchingHermesVersion` does not support `HERMES_V1_VERSION_NAME` and legacy Hermes.

From 0.83 onwards Hermes version is different from react-native version

File: packages/react-native/sdks/hermes-engine/hermes-utils.rb (hunk 14 above)
Comment (gabrieldonadel) [reply packages/react-native/sdks/hermes-engine/hermes-utils.rb:332]: Same thing, this does not support Hermes v1 and from my understanding is no longer necessary

File: packages/react-native/sdks/hermes-engine/version.properties (hunk 15 above)
Comment (gabrieldonadel) [reply packages/react-native/sdks/hermes-engine/version.properties:1]: we need to keep track of both versions somehow, not sure what you prefer but we can't just read the react-native version anymore

File: packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift (hunk 10 above)
Comment (gabrieldonadel) [reply packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift:13]: Couldn't find a way to explicitly import RCTUIKit.h

File: packages/react-native/package.json (hunk 16 above)
Comment (gabrieldonadel) [reply packages/react-native/package.json:202]: Using 0.0.0 causes pod install to fail

image

File: packages/react-native/React/Base/RCTUtils.mm (hunk 3 above)
Comment (gabrieldonadel) [reply packages/react-native/React/Base/RCTUtils.mm]: Done

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm (hunk 7 above)
Comment (gabrieldonadel) [reply packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm]: Fixed

File: packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm (hunk 4 above)
Comment (gabrieldonadel) [reply packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm:772]: Fixed

File: packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm (hunk 5 above)
Comment (Saadnajmi) [reply packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm:1767]: I'll leave this comment open so I remember to take extra care to test, since focus is quite important on desktop and the core refactor may have missed some edge cases we have

File: packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift (hunk 9 above)
Comment (Saadnajmi) [reply packages/react-native/ReactApple/RCTSwiftUI/RCTSwiftUIContainerView.swift:23]: On main, I separated RCTUIKit out to a new module that should be importable in Swift. I'll leave this comment open till we get to a revase with that module
